chromium/preloadlist/preloadlist.go

```diff
@@ -47,6 +47,33 @@ type PreloadList struct {
 	Entries []Entry `json:"entries"`
 }
 
+// PolicyType represents the policy under which the domain is a part of the preload list
+type PolicyType string 
+
+// Possible PolicyType values are as defined by 
+// https://source.chromium.org/chromium/chromium/src/+/main:net/http/transport_security_state_static.json
+const (
+	// test domains
+	Test PolicyType = "test"
+	// Google-owned sites
+	Google = "google"
+	// Entries without includeSubdomains or with HPKP
+	Custom = "custom"
+	// Bulk entries preloaded before Chrome 50
+	BulkLegacy = "bulk-legacy"
+	// Bulk entries with max-age >= 18 weeks (Chrome 50-63)
+	Bulk18Weeks = "bulk-18-weeks"
+	// Bulk entries with max-age >= 1 year (after Chrome 63)
+	Bulk1Year = "bulk-1-year"
+	// Public suffixes (e.g. TLDs or other public suffix list entries) preloaded at the owner's request.	
+	PublicSuffix = "public-suffix"
+	// domains under a public suffix that have been preloaded at the request of the the public suffix owner
+	// (e.g. the registry for the TLD).
+	PublicSuffixRequested = "public-suffix-requested"
+	// domains with an unspecified policy type
+	UnspecifiedPolicyType = ""
```

Comment:
Optional Nit: I don't think it's a hard requirement in the style guide, but usually the unspecified value is listed as the first one (to match int enums where it would be zero).

Reply:
Done.